app/models/validations/household_validations.rb

```diff
@@ -25,29 +25,29 @@ def validate_reasonable_preference(record)
 
   def validate_reason_for_leaving_last_settled_home(record)
     if record.reason == 32 && record.underoccupation_benefitcap != 4
-      record.errors.add :underoccupation_benefitcap, I18n.t("validations.household.underoccupation_benefitcap.dont_know_required")
-      record.errors.add :reason, I18n.t("validations.household.underoccupation_benefitcap.dont_know_required")
+      record.errors.add :underoccupation_benefitcap, I18n.t("validations.lettings.household.underoccupation_benefitcap.dont_know_required")
+      record.errors.add :reason, I18n.t("validations.lettings.household.underoccupation_benefitcap.dont_know_required")
```

Comment:
Can we split this per question?
 `I18n.t("validations.lettings.household.reason.dont_know_required")`